ui/components/multichain/account-picker/index.scss

```diff
@@ -0,0 +1,6 @@
+.multichain-account-picker {
+  &:hover,
+  &:focus {
+    background: var(--color-background-default-hover);
```

Comment:
This doesn't seem to be working for some reason? @garrettbear is it an issue with ButtonBase?

<img width="1440" alt="Screenshot 2023-03-18 at 11 07 20 AM" src="https://user-images.githubusercontent.com/8112138/226126049-2a0f226b-72e3-417b-a75f-b28e7dc8b827.png">

Reply:
It's faint but I'm definitely seeing the background change.